src/Test.FunctionalTests.Sarif/SarifLogEqualityComparerTests.cs

```diff
@@ -23,7 +24,7 @@ public class SarifLogEqualityComparerTests
         [Trait(TestTraits.Bug, "https://github.com/microsoft/sarif-sdk/issues/1046")]
         public void ValueEquals_ReturnsTrueForTwoIdenticalLogObjects()
         {
-            const string ComprehensiveTestSamplePath = @"v2\SpecExamples\Comprehensive.sarif";
+            string ComprehensiveTestSamplePath = Path.Combine(Environment.CurrentDirectory, "v2", "SpecExamples", "Comprehensive.sarif");
```

Comment:
>Combine [](http://example.com/codeflow?start=54&length=7)

why did you make this a full path?

Reply:
you are correct full path is not needed, fixed